charts/operator/templates/role.yaml

```diff
@@ -83,7 +83,7 @@ rules:
   - deployments
   apiGroups: ["apps"]
   resourceNames: ["rule-evaluator"]
-  verbs: ["get", "list", "watch"]
+  verbs: ["get", "list"]
```

Comment:
I think we should be able to get rid of them all? Any other places/reason we look them up?